### PR TITLE
[ACA-4355][ADW] The Process Management section becomes invisible on logout

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/ACA-4355

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by mauriziovitale — APPROVED** [review]
As we discussed the provider property is not needed. We already use the rule to decide if to how or not the section.

File: src/app/components/sidenav/sidenav.component.html [hunk 1]
```diff
@@ -7,23 +7,23 @@
...
       </ng-container>
 
       <ng-container *ngSwitchCase="'collapsed'">
         <div class="list-item" *ngFor="let item of group.items; trackBy: trackById">
-          <ng-container *ngIf="isLoggedIn(item.provider) && !item.component">
+          <ng-container *ngIf="!item.component">
             <app-button-menu [item]="item"></app-button-menu>
           </ng-container>
 
-          <ng-container *ngIf="isLoggedIn(item.provider) && item.component">
```
Comment (eromano) [inline src/app/components/sidenav/sidenav.component.html:26]: I don't think this modify will be ok when APS is not installed with ADW

**Review by eromano — CHANGES_REQUESTED** [review]

File: src/app/components/sidenav/sidenav.component.html (hunk 1 above)
Comment (sivakumar414ram) [reply src/app/components/sidenav/sidenav.component.html:26]: I feel the "LoggedIn" method is not required here, we already have custom rule to check same behaviour.
https://github.com/Alfresco/alfresco-digital-workspace-app/blob/develop/libs/content-ee/process-services-cloud-extension/assets/process-services-cloud.extension.json#L147

Comment (eromano) [reply src/app/components/sidenav/sidenav.component.html:26]: Alright

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**